src/vector/jitsi/index.ts

```diff
@@ -118,12 +119,25 @@ function joinConference() { // event handler bound in HTML
     if (avatarUrl) meetApi.executeCommand("avatarUrl", avatarUrl);
     if (userId) meetApi.executeCommand("email", userId);
 
-    meetApi.on("readyToClose", () => {
-        switchVisibleContainers();
+    let meetingClosed = new Promise(resolve => {
```

Comment:
Without looking, I think this is what the linter is complaining about.
```suggestion
    const meetingClosed = new Promise(resolve => {
```